content/dstu2/patient.md

```diff
@@ -22,7 +42,8 @@ Search for Patients that meet supplied query parameters:
 
 _Implementation Notes_
 
-* The [Patient.animal] and [Patient.link] modifier elements are not supported and will not be returned.
+* The [Patient.animal] modifier element is not supported and will not be returned.
+* When using an OAuth 2 patient access token to call GET /Patient without parameters, a bundle will be returned which contains the patients listed in the access token.
```

Comment:
Clarify: If the current user is a patient or patient proxy, a searches can be done without any parameters. The search will return all patients the current user has been granted access to view.j

Is there a scope restriction on this (like there is on provider)? Or would this work for either patient/Patient.read and user/Patient.read?